[Bug] Pod studio-frontend Crashing with MODULE_NOT_FOUND Error

### Priority

Undecided

### OS type

Ubuntu

### Hardware type

Xeon-other (Please let us know in description)

### Installation method

- [ ] Pull docker images from hub.docker.com
- [X] Build docker images from source

### Deploy method

- [ ] Docker compose
- [ ] Docker
- [X] Kubernetes
- [ ] Helm

### Running nodes

Single Node

### What's the version?

Commit Id - 62a35ea8ef086fef050dbeeb4208de7aeb2f396a

### Description

The studio-frontend pod is consistently crashing upon startup. Below are the logs extracted from the pod, which indicate a MODULE_NOT_FOUND error.

**Hardware**: Intel(R) Xeon(R) Platinum 8480+

### Reproduce steps

1) Deploy the application in the studio namespace by running the provided Ansible playbook.
**command**: **_ansible-playbook -i inventory deploy_studio.yml_**
2) Observe the pod studio-frontend failing to initialize.
3) Run kubectl logs studio-frontend -n studio to see the logs.

### Raw log

```shell
> flowise@2.1.1 start /usr/src
> run-script-os

> flowise@2.1.1 start:default
> cd packages/server/bin && ./run start

(node:41) [MODULE_NOT_FOUND] ModuleLoadError Plugin: flowise: [MODULE_NOT_FOUND] require failed to load /usr/src/packages/server/dist/commands/start.js: Cannot find module '/usr/src/node_modules/.pnpm/@langchain+openai@0.3.13_@langchain+core@0.2.18_openai@4.51.0_encoding@0.1.13___encoding@0.1.13/node_modules/openai/helpers/zod.js'
module: @oclif/core@1.26.2
task: toCached
plugin: flowise
root: /usr/src/packages/server
See more details with DEBUG=*
(Use `node --trace-warnings ...` to show where the warning was created)
 ›   Error: command start not found
 ELIFECYCLE  Command failed with exit code 2.
```

--- a/studio-frontend/package.json
+++ b/studio-frontend/package.json
@@ -68,7 +68,7 @@
     "resolutions": {
         "@qdrant/openapi-typescript-fetch": "1.2.6",
         "@google/generative-ai": "^0.15.0",
-        "openai": "4.51.0",
+        "openai": "4.57.3",
         "@langchain/core": "0.2.18",
         "axios": "1.7.4",
         "nth-check": "2.0.1",

--- a/studio-frontend/packages/server/package.json
+++ b/studio-frontend/packages/server/package.json
@@ -76,7 +76,7 @@
         "moment-timezone": "^0.5.34",
         "multer": "^1.4.5-lts.1",
         "mysql2": "^3.9.2",
-        "openai": "^4.51.0",
+        "openai": "^4.57.3",
         "pg": "^8.11.1",
         "posthog-node": "^3.5.0",
         "reflect-metadata": "^0.1.13",

